src/mbgl/renderer/buckets/fill_extrusion_bucket.cpp

```diff
@@ -102,9 +102,6 @@ void FillExtrusionBucket::addFeature(const GeometryTileFeature& feature,
 
                     const Point<double> perp = util::unit(util::perp(d1 - d2));
                     const auto dist = util::dist<int16_t>(d1, d2);
-                    if (dist > std::numeric_limits<int16_t>::max()) {
-                        edgeDistance = 0;
-                    }
```

Comment:
Same here?

Reply:
Hmm yeah, but I think this means that we should change the underlying code that leads to `dist` being a `uint16_t`, since the overflow will be hidden there.

Reply:
followup ticket 👍

Reply:
https://github.com/mapbox/mapbox-gl-native/issues/11909